ansible/config_sbps_iscsi_targets.yml

```diff
@@ -42,4 +42,4 @@
     # Configure SBPS DNS "SRV" and "A" records
     - role: csm.sbps.dns_srv_records
     # Mount s3 bucket 'boot-images' using s3fs read-only policy for SBPS agent
-    - role: csm.sbps.mount_s3_images
+    #- role: csm.sbps.mount_s3_images
```

Comment:
We don't want to check in commented lines. If this no longer is needed, we should just remove the line.

```suggestion
```